photon-ml/src/test/scala/com/linkedin/photon/ml/cli/game/EvaluatorParamsTest.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.cli.game
+
+import org.testng.Assert._
+import org.testng.annotations.Test
+
+import com.linkedin.photon.ml.evaluation._
+
+class EvaluatorParamsTest {
+
+  @Test
+  def precisionAtKIdTypeSetTest(): Unit = {
+    val expectedIdTypeSet = Set("documentId", "queryId", "foo", "bar")
+    val precisionAtKEvaluators = expectedIdTypeSet.toSeq.flatMap(t => Seq(1, 3, 5, 10).map(PrecisionAtK(_, t)))
+    val allEvaluators = Seq(AUC, RMSE, LogisticLoss) ++ precisionAtKEvaluators
+
+    val evaluatorParamsMocker = new EvaluatorParams {}
```

Comment:
Looks fine to me. You don't really want to mock this object, because you're testing one of its methods. Also, a spy would have the same issue, since you'd need to create the object first.